src/openapi.yaml

```
@@ -6728,6 +6688,239 @@ components:
+            in de statuten (rechtspersoon) of in de vennootschapsovereenkomst is overeengekomen
+            (Vennootschap onder firma of Commanditaire vennootschap).
+          type: string
+          maxLength: 500
+        rechtsvorm:
+          title: Rechtsvorm
+          description: De juridische vorm van de NIET-NATUURLIJK PERSOON.
+          type: string
+          enum:
+          - Besloten Vennootschap
+          - Cooperatie, Europees Economische Samenwerking
+          - Europese Cooperatieve Venootschap
+          - Europese Naamloze Vennootschap
+          - Kerkelijke Organisatie
+          - Naamloze Vennootschap
+          - Onderlinge Waarborg Maatschappij
+          - Overig privaatrechtelijke rechtspersoon
+          - Stichting
+          - Vereniging
+          - Vereniging van Eigenaars
+          - Publiekrechtelijke Rechtspersoon
+          - Vennootschap onder Firma
+          - Maatschap
+          - Rederij
+          - Commanditaire vennootschap
+          - Kapitaalvennootschap binnen EER
+          - Overige buitenlandse rechtspersoon vennootschap
+          - Kapitaalvennootschap buiten EER
+        bezoekadres:
+          title: Bezoekadres
+          description: De gegevens over het adres van de NIET-NATUURLIJK PERSOON
+          type: string
+          maxLength: 1000
+        subVerblijfBuitenland:
+          title: Sub verblijf buitenland
+          description: De gegevens over het verblijf in het buitenland
+          type: string
+          maxLength: 1000
+    betrokkene_identificatie_RolNietNatuurlijkPersoon:
+      type: object
+      properties:
+        betrokkeneIdentificatie:
+          $ref: '#/components/schemas/RolNietNatuurlijkPersoon'
+    RolVestiging:
+      title: betrokkene_identificatie
+      type: object
+      properties:
+        vestigingsNummer:
+          title: Vestigings nummer
+          description: Een korte unieke aanduiding van de Vestiging.
+          type: string
+          maxLength: 24
+        handelsnaam:
+          description: De naam van de vestiging waaronder gehandeld wordt.
+          type: array
+          items:
+            title: Handelsnaam
+            type: string
+            maxLength: 625
+        verblijfsadres:
+          title: Verblijfsadres
+          description: De gegevens over het verblijf en adres van de Vestiging
+          type: string
+          maxLength: 1000
+        subVerblijfBuitenland:
+          title: Sub verblijf buitenland
+          description: De gegevens over het verblijf in het buitenland
+          type: string
+          maxLength: 1000
+    betrokkene_identificatie_RolVestiging:
+      type: object
+      properties:
+        betrokkeneIdentificatie:
+          $ref: '#/components/schemas/RolVestiging'
+    Vestiging:
+      type: object
+      allOf:
+      - $ref: '#/components/schemas/Rol'
+      - $ref: '#/components/schemas/betrokkene_identificatie_RolVestiging'
+    RolOrganisatorischeEenheidSerialezer:
```

Think there's a typo in `vng-api-common`, this happens a couple of times

```suggestion
    RolOrganisatorischeEenheidSerializer:
```

---

fixed typo mistake in `RolOrganisatorischeEenheidSerializer` name